widget/scroller.go

```diff
@@ -279,6 +279,10 @@ func (r *scrollContainerRenderer) MinSize() fyne.Size {
 }
 
 func (r *scrollContainerRenderer) Refresh() {
+	if len(r.BaseRenderer.Objects()) == 0 || r.BaseRenderer.Objects()[0] != r.scroll.Content {
+		// push updated content object to baseRenderer
+		r.BaseRenderer.SetObjects([]fyne.CanvasObject{r.scroll.Content})
+	}
```

Comment:
This bug fix could be merged separately

Reply:
It was, as 2ccac85f6, this is just due to the base being old I think?